pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -223,6 +223,12 @@ class ChangeFeedTestCase(BaseAPITestCase):
     <https://pulp.plan.io/issues/1922>`_.
     """
 
+    @classmethod
+    def setUpClass(cls):
+        """Set config used by each test case."""
```

Comment:
Minor: What about update the docstring? It is creating more than config variable.